src/pages/CreateCommunity/Components/ConnectedTokens.tsx

```diff
@@ -0,0 +1,292 @@
+import React, { useState, useEffect } from 'react';
+import { CheckIcon, SelectorIcon, TrashIcon } from '@heroicons/react/solid';
+import { Combobox } from '@headlessui/react';
+import ClassNamesLogic from 'components/Util/ClassNamesLogic';
+import { useRecoilValue } from 'recoil';
+import { userInfoAtom } from 'services/recoil/userInfo';
+import { IfungibleToken, InonFungibleTokenCollection } from 'types/types';
+import { useTokenRegistry } from 'components/Solana/TokenRegistry';
+import solanaLogo from 'assets/solanaLogo.svg';
+import defaultUserProfileImage from 'assets/defaultUserProfileImage.png';
+import {
+  GetOnChainNftMetadata,
+  GetOffChainNftMetadata,
+} from 'services/Solana/GetData/GetTokenData';
+
+type TtokenList = {
```

Comment:
This type should be TtokenListItem, otherwise token list makes it seem like its a list.

Reply:
okk